test/JDBC/expected/BABEL-2449-vu-prepare.out

```diff
@@ -168,3 +168,39 @@ BEGIN
 RETURN (SELECT DATETIMEOFFSETFROMPARTS(2011.9, 8.9, 15.9, 14.9, 30.9, 00, 500.9, 12.9, 30.9, 3));
 END
 GO
+
+CREATE FUNCTION BABEL_2449_vu_prepare_f14()
+RETURNS DATETIMEOFFSET AS
+BEGIN
+RETURN (SELECT DATETIMEOFFSETFROMPARTS(1700, 8, 15, 14, 30, 00, 500, 12, 30, 3));
+END
+GO
+
+CREATE FUNCTION BABEL_2449_vu_prepare_f15()
+RETURNS DATETIMEOFFSET AS
+BEGIN
+RETURN (SELECT DATETIMEOFFSETFROMPARTS(NULL, 8, 15, 14, 30, 00, 500, 12, 30, 3));
+END
+GO
+
+CREATE FUNCTION BABEL_2449_vu_prepare_f16()
+RETURNS DATETIMEOFFSET AS
+BEGIN
+RETURN (SELECT DATETIMEOFFSETFROMPARTS(1, 8, 15, 14, 30, 00, 500, 12, 30, 3));
+END
+GO
+
+CREATE FUNCTION BABEL_2449_vu_prepare_f17()
+RETURNS DATETIMEOFFSET AS
+BEGIN
+RETURN (SELECT DATETIMEOFFSETFROMPARTS(0*1, 8, 15, 14, 30, 00, 500, 12, 30, 3));
+END
+GO
+
+CREATE FUNCTION BABEL_2449_vu_prepare_f18()
+RETURNS DATETIMEOFFSET AS
+BEGIN
+RETURN ( Select DATETIMEOFFSETFROMPARTS(1, 1, 1, 4, 30, 00, 500, 12, 30, 3));
```

Comment:
why should this fail?

Reply:
As datetimeoffset lower limit is `0001-01-01 00:00:00.000 +00:00`  and the above time is 
`0001-01-01 04:30:00.500 +12:30` , here timezone offset is +12:30 which is not possible given the lower limit.